galley/cmd/galley/cmd/root.go

```diff
@@ -42,7 +43,7 @@ var (
 )
 
 // GetRootCmd returns the root of the cobra command-tree.
-func GetRootCmd(args []string, printf, fatalf shared.FormatFn) *cobra.Command {
+func GetRootCmd(args []string, getenv func(string) string, printf, fatalf shared.FormatFn) *cobra.Command {
```

Comment:
FYI, your PR will conflict with some other PR that removes printf & fatalf etc. You can simply call getenv from within the code under cmd/